src/birl.gleam

```diff
@@ -300,7 +300,8 @@ pub fn parse(value: String) -> Result(Time, Nil) {
   {
     [day_string, time_string], _, _
     | _, [day_string, time_string], _
-    | _, _, [day_string, time_string] -> Ok(#(day_string, time_string))
+    | _, _, [day_string, time_string]
+    -> Ok(#(day_string, time_string))
```

Comment:
@giacomocavalieri Does this formatting look right to you? Or is this a bug in the formatter?

Reply:
Mmh this look like a bug, could you please open an issue on the gleam repo? (Maybe tag me there too, so I don't forget about this 😁)

Reply:
Opened as https://github.com/gleam-lang/gleam/issues/3136.